packages/shared/src/features/onboarding/steps/FunnelLoading.tsx

```diff
@@ -142,53 +139,47 @@ const FunnelLoading = ({
   };
 
   return (
-    <FunnelStepBackground variant={FunnelBackgroundVariant.Hourglass}>
-      <div className="flex min-h-dvh flex-col items-center justify-center gap-10 px-1 mobileL:px-6">
-        <div className="relative">
-          <svg
-            className="size-48"
-            viewBox="0 0 196 196"
+    <div className="flex min-h-dvh flex-col items-center justify-center gap-10 px-1 mobileL:px-6">
+      <div className="relative">
+        <svg
+          className="size-48"
+          viewBox="0 0 196 196"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          {/* Background circle */}
+          <circle
+            cx="98"
+            cy="98"
+            r="90"
+            className="stroke-overlay-primary-pepper"
```

Comment:
This is the right color it's solid non themed one?

Reply:
Don't know, this comes from @AmarTrebinjac component, just removed the bg wrapper